modules/debts_api/lib/debts_api/v0/one_debt_letter_service.rb

```diff
@@ -3,205 +3,216 @@
-            line_number = debt_index + 1
-            table_data << [
-              "#{line_number}.  #{debt['benefitType']}" || '',
-              "$#{format_amount(debt['currentAR'] || 0)}", # Plain string
-              ''
-            ]
-          end
+          if debts.any?
+            table_data = [['Benefits Overpayment', 'AMOUNT DUE', '']]
+            table_data << [{ content: BENEFITS_TABLE_DESCRIPTION, inline_format: true }, '', '']
+            debts.each_with_index do |debt, debt_index|
+              line_number = debt_index + 1
+              table_data << [
+                "#{line_number}.  #{debt['benefitType']}" || '',
+                "$#{format_amount(debt['currentAR'] || 0)}", # Plain string
+                ''
+              ]
+            end
 
-          total_debts = debts.sum { |d| (d['currentAR'] || 0).to_f }
-          formatted_debt_total = "$#{format_amount(total_debts)}"
-          table_data << [DEBT_TOTAL_TEXT, formatted_debt_total, '']
-          table_data << [{ content: DEBT_PAYMENT_INSTRUCTIONS, inline_format: true }, '', '']
-          pdf.table(table_data, width: pdf.bounds.width, cell_style: { padding: 5, size: 8 }) do
-            cells.borders = %i[left right]
-            row(0).columns(0).style(font_style: :bold_italic)
-            rows(0).borders = %i[top bottom left right]
-            rows(-1).borders = %i[bottom left right]
-            rows(-2).columns(0).align = :right
-            column(1).align = :right
+            total_debts = debts.sum { |d| (d['currentAR'] || 0).to_f }
+            formatted_debt_total = "$#{format_amount(total_debts)}"
+            table_data << [DEBT_TOTAL_TEXT, formatted_debt_total, '']
+            table_data << [{ content: DEBT_PAYMENT_INSTRUCTIONS, inline_format: true }, '', '']
+            pdf.table(table_data, width: pdf.bounds.width, cell_style: { padding: 5, size: 8 }) do
+              cells.borders = %i[left right]
+              row(0).columns(0).style(font_style: :bold_italic)
+              rows(0).borders = %i[top bottom left right]
+              rows(-1).borders = %i[bottom left right]
+              rows(-2).columns(0).align = :right
+              column(1).align = :right
+            end
+          else
+            pdf.text 'No overpayments due.', size: 10
           end
-        else
-          pdf.text 'No overpayments due.', size: 10
-        end
-      end.render
+        end.render
 
-      legalese_pdf = load_legalese_pdf
-      combined_pdf = CombinePDF.parse(debt_letter_pdf) << legalese_pdf
+        legalese_pdf = load_legalese_pdf
+        combined_pdf = CombinePDF.parse(debt_letter_pdf) << legalese_pdf
 
-      combined_pdf.to_pdf
-    end
-    # rubocop:enable Metrics/MethodLength
+        combined_pdf.to_pdf
+      end
 
-    def save_pdf(filename = default_filename)
-      save_pdf_content(filename, get_pdf)
-      filename
-    end
+      # rubocop:enable Metrics/MethodLength
 
-    private
+      def combine_pdfs(document)
```

Comment:
ATTN: reviewer

This is really the only change in this file along with the optional argument in get_pdf.

We were getting an uninitilized constant error with the way the class was defined. Changing it got rid of the error.